google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITTransactionManagerAsyncTest.java

```diff
@@ -224,6 +222,8 @@ public void testRollback() throws InterruptedException {
     }
   }
 
+  @Ignore(
+      "Cloud Spanner now seems to return CANCELLED instead of ABORTED when a transaction is invalidated by a later transaction in the same session")
```

Comment:
This seems to be a slight behavioral change in Cloud Spanner: Starting a new read/write transaction on a session that already has a read/write transaction will invalidate the first transaction. As far as I can tell, that would previously return an `ABORTED` error. That seems to have been changed to a `CANCELLED` error, which does make sense as it is an indication of an application error rather than a transient server error.